The command-line tool does not support absolute paths

**Install source and version**
- [ ] Installed from the Windows Store
- [ ] Installed with the MSIX from python\.org
- [ ] Installed with the MSI from python\.org
- [ x] Installed with `winget install 9NQ7512CXL7T`

Version: 25.0b14


**Describe the bug**
Absolute paths cannot be used in the command-line tool.

**To Reproduce**

Use absolute paths in the "target" or "download" parameters within the command line arguments

```bat
pymanager install 3.12-32 "--target=I:\kuankuan\Download\test-pym"
> [ERROR] Unexpected argument: --target=I

pymanager install 3.12-32 --target=I:\kuankuan\Download\test-pym
> [ERROR] Unexpected argument: --target=I

pymanager install 3.12-32 --target="I:\kuankuan\Download\test-pym"
> [ERROR] Unexpected argument: --target=I

pymanager install 3.12-32 --download=I:\kuankuan\Download\test-pym
> [ERROR] Unexpected argument: --download=I
```

**Expected behavior**
Download or extract this version of Python to `I:\kuankuan\Download\test-pym`

--- a/src/manage/commands.py
+++ b/src/manage/commands.py
@@ -373,7 +373,11 @@ class BaseCommand:
                 seen_cmd = True
             elif a.startswith(("-", "/")):
                 a, sep, v = a.partition(":")
-                if not sep:
+                if sep:
+                    if "=" in a:
+                        a, sep, v_pre = a.partition("=")
+                        v = f"{v_pre}:{v}"
+                else:
                     a, sep, v = a.partition("=")
                 set_next = a.lstrip("-/").lower()
                 try:

--- a/tests/test_commands.py
+++ b/tests/test_commands.py
@@ -81,3 +81,25 @@ def test_legacy_listpaths_command_help(assert_log, patched_installs):
     assert_log(
         assert_log.skip_until(r".*List command.+py list.+"),
     )
+
+
+def test_install_command_args():
+    # This is not meant to be exhaustive test of every possible option, but
+    # should cover all of the code paths in BaseCommand.__init__.
+    for args in [
+        ["-v", "-y"],
+        ["--v", "--y"],
+        ["/v", "/y"],
+    ]:
+        cmd = commands.InstallCommand(args)
+        assert cmd.log_level == logging.VERBOSE
+        assert not cmd.confirm
+
+    for args in [
+        ["--log", "C:\\LOG.txt"],
+        ["/log", "C:\\LOG.txt"],
+        ["--log:C:\\LOG.txt"],
+        ["--log=C:\\LOG.txt"],
+    ]:
+        cmd = commands.InstallCommand(args)
+        assert cmd.log_file == "C:\\LOG.txt"

